frontend/app/forms/inputs/text-input-group.component.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { useCss } from "kremling";
+import { DataContainerContext } from "../data-container.component";
+
+export default function TextInputGroup(props) {
+  const dataContext = React.useContext(DataContainerContext);
+  var inputs = Object.assign({}, ...props.inputs);
+  const [inputList, setInputList] = React.useState([inputs]);
+  const scope = useCss(css);
+
+  return (
+    <div className="web-form-input" onBlur={evt => handleBlur(evt)} {...scope}>
+      <label>{props.groupLabel}</label>
+      <div>
+        {inputList.map((input, idx) => (
+          <React.Fragment key={`${input}-${idx}`}>
+            {Object.entries(input).map(([key, val]) => (
+              <div key={`${key}-${idx}`} className="web-form-input text-input">
+                <label>{key}</label>
```

Comment:
For better accessibility, we should connect this label with the input it's labeling. Easiest way might be to put a unique id on the inputs and then do `<label for="the-id">`. Since there are multiple inputs, maybe using the index in the array as part of the id to guarantee uniqueness.